rpcd/playbooks/roles/rpc_maas/tasks/ensure_filesystem_checks.yml

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-- name: Install fileystem auto checks
+- name: Install fileystem checks
```

Comment:
Thats on purpose to because we can differentiate between auto provisioned filesystem checks and manual, by overrides